java/pom.xml

```diff
@@ -155,8 +168,12 @@
                 </executions>
                 <configuration>
                     <redirectTestOutputToFile>true</redirectTestOutputToFile>
-                    <forkMode>once</forkMode>
+
+                    <!-- It's important to run tests serially because we are doing timing tests in some of the tests. -->
```

Comment:
So, currently, it should spawn one thread and reuse for all tests as far as I can tell.  This works fine.  What was hanging intermittently was the use of `<forkMode>once</forkMode>`.  My suspicion is that it has something to do with calling `VWLearners.create` many times concurrently.  That's just a guess though.